assimilation_code/modules/observations/atmosphere_quantities.f90

```diff
@@ -0,0 +1,197 @@
+! DART software - Copyright UCAR. This open source software is provided
```

Comment:
Are the units documented anywhere?

Reply:
nope.  huge can of worms.  run away.   (seriously, i think the 'udunits' package that netcdf and other programs use is probably a good starting point for names of units.)

Reply:
Todo: what should go in the documentation about the units column